pipelines/beamform.nf

```diff
@@ -184,9 +184,20 @@ process do_beamform {
         fi
 
         apptainer exec -B /fred/oz313/:/fred/oz313/ $params.container bash -c 'source /opt/setup_proc_container && python3 $beamform_dir/craftcor_tab.py \$args'
-        rm TEMP*
```

Comment:
I guess this is not needed because those write/read steps aren't happening any more?

Reply:
correct